jobqueue/scheduler/lsf.go

```diff
@@ -321,13 +320,13 @@ func (s *lsf) initialize(ctx context.Context, config interface{}) error {
 	// prefer the queue that is more limited, since we suppose they might be
 	// less busy or will at least become free sooner
 	criteriaHandling := map[string][]int{
-		"hosts":      {20, 1}, // weight, sort order
-		"max_user":   {16, 1},
-		"max":        {15, 1},
-		"prio":       {4, 1},
+		"hosts":      {18, 1}, // weight, sort order
+		"max_user":   {10, 1},
+		"max":        {5, 1},
+		"prio":       {5, 0},
 		"chunk_size": {10000, 0},
-		"runlimit":   {2, 0},
-		"memlimit":   {2, 0},
+		"runlimit":   {5, 1},
```

Comment:
I'm a bit concerned that you had to change the sort order on 2 of these. Did it make logical sense to you that this was the correct thing to do?

Reply:
Changing the runlimit to be {5, 1} means a higher runlimit is favoured, the original way meant that the gpu queues were ranked much higher than wanted, and the new way puts those gpu-queues much nearer the end.

The change for prio to {4, 0} means normal, long, parallel and week all rank much better as they all have lower prios. Without this, week ranks worse than several queues.